pkgs/by-name/sd/sdrpp/package.nix

```
@@ -125,25 +136,31 @@ stdenv.mkDerivation rec {
     uhd
     boost
   ]
-  ++ lib.optional audio_sink rtaudio
+  ++ lib.optional (audio_source || audio_sink) rtaudio
   ++ lib.optional portaudio_sink portaudio
+  # ++ lib.optional (dab_decoder || m17_decoder) codec2;
   ++ lib.optional m17_decoder codec2;
```

I think you forgot to exchange those lines. Nothing critical, as we discussed before, dab is broken, but uncommenting should keep it consistent.

```suggestion
  ++ lib.optional (dab_decoder || m17_decoder) codec2;
```